### DISCUSSION TITLE
The construction of the coliseum, sanctuary and hydropome is very slow. Even with a lot of architects and field workers.

### DISCUSSION THREAD

**Discussion by Edu2014** [Keriew/augustus#475] [General]
I put 10 guild of architects and field workers and it still takes too long. Wouldn't it be better faster?

**Comment by Edu2014** [comment]
I don't know how I can change this in mod files.

**Comment by Areldir** [comment]
Hi there!

Monument construction is intentionally quite slow. The best way to speed the process is to have a large quantity of the required resources sitting in warehouses built close to both workcamps and the monument itself. That way, you minimize the travel time of workers between their guild, the warehouse and the monument.

**Comment by gianlore** [comment]
Anche se metti 100 gilde, la velocità di costruzione non aumenterà. E non credo aumenti neppure costruendo più campi di lavoro e la cosa mi sembra giusta.
----
Even if you put 100 guilds, the construction speed will not increase. And I don't think it increases even by building more work camps and it seems right to me.
